apps/studio/prisma/scripts/migrations/addOrdering/addOrdering.ts

```diff
@@ -0,0 +1,89 @@
+import { DEFAULT_CHILDREN_PAGES_BLOCK } from "@opengovsg/isomer-components"
+
+import { db, sql } from "~/server/modules/database"
+import { updateBlobById } from "~/server/modules/resource/resource.service"
+
+export const up = async () => {
+  // Step 1: select all index pages and their blobs where
+  // the text `"type": "childrenpages"` isn't found
+  const sqlQuery = sql<{
+    id: string
+    siteId: string
+    content: PrismaJson.BlobJsonContent
+  }>`
+    SELECT
+      *
+    FROM
+      (
+        SELECT
+          COALESCE("db"."content", "pb"."content") AS "content",
+          "Resource"."id", 
+          "Resource"."siteId"
+        FROM
+          "Resource"
+          LEFT JOIN "Version" ON "Version"."id" = "Resource"."publishedVersionId"
+          LEFT JOIN "Blob" AS "pb" ON "pb"."id" = "Version"."blobId"
+          LEFT JOIN "Blob" AS "db" ON "db"."id" = "Resource"."draftBlobId"
+        WHERE
+          "type" = 'IndexPage'
+      ) AS "indexContent"
+    WHERE
+      ("content"->'content')::text ILIKE '%"type": "childrenpages"%' and "content" ->> 'layout' = 'index' and ("content" -> 'content')::text NOT ILIKE '%"childrenPagesOrdering"%';
+`
+
+  const result = await sqlQuery.execute(db)
+
+  for (const r of result.rows) {
+    const last = r.content.content.at(-1)
+
+    // Skip if the last block is not a childrenpages block
+    if (last?.type !== "childrenpages") {
+      console.log(
+        `Skipping Resource ${r.id}: last block is not childrenpages (type: ${last?.type})`,
+      )
+      continue
+    }
```

Comment:
question: do we need to find for the `childrenpages` block to add the `childrenPagesOrdering` property? Seems like potentially there can be pages that are missing that.